### DISCUSSION TITLE
因 docker 未知 bug 导致首次启动后界面报错 “.....invalid IPv6 address in resolver ......” 的解决办法

### DISCUSSION THREAD

**Discussion by forxiucn** [chaitin/SafeLine#689] [Q&A] [answered]
### 问题描述

错误内容：2024/02/07 11:21:56 [INFO] config/config.go:22 nginx conf path: /etc/nginx/sites-enabled stream: false
nginx: [emerg] invalid IPv6 address in resolver "[f*::*:*:*:*%enp1s0]" in /etc/nginx/nginx.conf:132

### 版本号

4.3.2

### 复现方法

安装即可发现

### 期望的结果

可正常启动

**Comment by dong4j** [comment, marked as answer]
编辑寄主机的 /etc/resolv.conf, 删除 ipv6 的 nameserver

**Comment by delong8** [comment]
> 编辑寄主机的 /etc/resolv.conf, 删除 ipv6 的 nameserver

正解。 docker 在创建虚拟网络时，会在操作系统上注册 IPv6 的解析地址，因为不知名原因，docker 会把 /etc/resolv.conf 改错，导致 IPv6 解析失败。这时候手动把系统配置文件中的错误内容删掉，就可以正常使用了
